source/User_Guide/Transactional_Templates/create_and _edit_dynamic_transactional_templates.md

```diff
@@ -0,0 +1,162 @@
+
+{% anchor h3 %}
+Using the Code Editor
+{% endanchor %}
+
+*To edit a template using the Code Editor:*
+
+1. If you selected the Code Editor, copy and paste your HTML code into the code editor. 
+<br></br>If you paste in an email template, it is likely that you will overwrite the default and very basic email HTML that is provided in the code editor. 
+2. To add an image to your template, select the **images** icon above the HTML code.
+3. Select the image you want to add from your image library. To add more images to your library, drag them into the box at the top of the library or click **choose images to upload**. 
+4. Select the **Image Details** tab and copy the URL in the *Image Source URL* tab by clicking **Copy URL**.
+5. Paste this URL in an image source tag in your campaign or template's HTML.
+Example: `<img src="your image URL here">`
+6. When you’re done editing, click **Save**. <br></br>Please double check that the content is formatted correctly before you send an email using this template.
+
+{% anchor h3 %}
+Using the Design Editor
+{% endanchor %}
+
+*To edit a template using the Design Editor:*
+
+{% anchor h2 %}
+Preview and test your template
+{% endanchor h2 %}
+
+SendGrid suggests previewing and testing your template once you have created it to ensure that the format and layout look the way you expect.
+
+*To preview your template:*
+ 
+1. Select **Preview**.  <br></br>A window pops out with a preview of the email that you’re going to test, with the option to view the template formatted for both mobile and desktop platforms.
+
+![](/images/template_engine_12.png "Preview option")
+
+1. Click **Send Test** to send a test email using your new template. 
+1. Fill in the *recipient* and *subject line* fields and then click **Send**.  
+
+![](/images/templates_create_edit_8.png "Preview and Test")
+
+![](/images/templates_create_edit_9.png "Send Test")
+
+Any Substitution tags will not be populated in this test, but you will be able to confirm the layout and format of your template.
+
+{% anchor h2 %}
+Activate your template
+{% endanchor h2 %}
+
+*To activate your template:*
+
+1. Navigate to the template you wish to use and select the action menu.
+1. Select **Make Active**.  
+
+{% info %}
+
+A template can only have one active version at a time. If you’ve created a new version with different HTML that you want your customers to start receiving, you’ll need to make that version “Active.”
+
+{% endinfo %}
+
+{% anchor h2 %}
+Duplicate a Template
+{% endanchor h2 %}
+
+*To duplicate a template:*
+
+1. Navigate to the template you wish to use and select the action menu.
+1. Select **Duplicate**.  
+
+This creates a clone, giving it a name that includes the original version name, the word “copy,” and today’s date. This duplicate version will not be active until you activate it.
+
+{% anchor h2 %}
+Unsubscribe Substitution Tags
```

Comment:
Remove this section and add a link to https://sendgrid.com/docs/User_Guide/Transactional_Email/unsubscribes.html. Then update the unsubscribes page with all the details you have below.